README.md

```diff
@@ -40,13 +40,16 @@ in the GitHub Docs.
 
 #### Using a subkey from an existing GPG key
 
-First open your master key for editing:
+First open your master key for editing (use `--list-keys` to find it):
 
 ```bash
 gpg --edit-key "<YOUR MASTER KEY ID>"
 ```
 
-Type `addkey` and select signing or s for capabilities. RSA key type is recommended for greatest compatibility.
+A signing key type RSA with 3072 bits is recommended for greatest compatibility.
```

Comment:
Discussed during other comments in the review: I'd say that the length should not be endorsed by us, since the tool already endorses one, while if we endorse it here it will become old knowledge pretty quickly.

Reply:
I thought to express it's as a recommendation as is RSA recommended here. Would "at least 3072" be better?

There's also a recommendation about expiry, which addresses that the security related recommendation can become "old knowledge" pretty soon, perhaps should change the order of the text?

I think more info here is better than "go search yourself", because this project could be used by very beginner users, and it has said the gpg stuff is PITA even for novice users.

Reply:
Would be best to not recommend a value nor provide any guidance for the key length at all: this target changes, and the first time a security researcher with a bad mood shows up we have a problem :-)

Reply:
For the same reasons I would not recommend any cryptosystem here. Whether people use RSA or DSA or any ElipticCurve or whatnot  is their preference and dependend on their toolset.

Reply:
Aye: the only requirement is that the key must be usable for signing :+1:

Reply:
Ok. let's remove any recommendation then, just "chose signing"